src/genie/libs/parser/iosxr/show_bgp.py

```diff
@@ -5062,7 +5063,7 @@ def cli(self, vrf_type='all', address_family='', instance='all', vrf='all', outp
 
         # 2219             0 200 33299 51178 47751 {27016} e
         p16_2 = re.compile(r'^\s*(?P<metric>[0-9]+) +(?P<weight>[0-9]+)'
-                           r' +(?P<path>[0-9\{\}\s]+) '
+                           r' +(?P<path>[0-9\.\{\}\s]+) '
```

Comment:
Can you please add a comment with an example that supports this change?

Reply:
I have added possible example, as the change was to capture dotted ASN.